src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectNode.Sorting.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I changed my mind halfway through implementation. I was initially leaning towards .Sorting.cs files but I don't want to end up with a million of these things.